app/src/main/java/co/nimblehq/compose/crypto/ui/theme/Style.kt

```diff
@@ -41,6 +41,17 @@ object Style {
         @Composable
         get() = if (isLight) SonicSilver else LightSilver
 
+    val Colors.coinInfoSellBuyBackground: Color
+        @Composable
+        get() = if (isLight) White else DarkJungleGreen
```

Comment:
I personally think it's just a design mistake, they could want to use the same color to the main screen background (see dark mode).

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/16315358/189338828-1a156268-6ab1-4e93-a830-ae3d2badb1e4.png">

Reply:
@luongvo Not sure about this but I think it looks good when having a white background on a light theme. 🤔 cc @hoangnguyen92dn

Reply:
@Wadeewee @luongvo I think we could follow the design, it seems with the white background these buttons highlighted a bit so please leave it as it is 😄 ✋